third_party/2and3/mock.pyi

```diff
@@ -0,0 +1,181 @@
+# Stubs for unittest.mock
+
+import sys
+from typing import Any, Optional, Type
+
+if sys.version_info >= (3, 3):
```

Comment:
This appears to be the problem - for Python 2 this is left as an empty module.

Reply:
Oops you're right. We should make it so that everything gets unconditionally exported. (We don't care about 3.2 anyway.)

Can you make a PR making that change?

Reply:
Submitted #2201